src/Symfony/Component/Console/Application.php

```diff
@@ -1044,6 +1052,18 @@ private function findAlternatives($name, $collection)
     public function setDefaultCommand($commandName)
```

Comment:
What about adding a new argument here that says it is the default but also the only command?